File: LISTADO DE PRÁCTICAS DE LA SEGUNDA EVALUACIÓN PARCIAL - LUIS ARTURO MONSIVAIS VARGAS - 18550667/TRABAJOS EN FLOWGORITH - JAVA - C++ - PYHTON/Códigos en C++/EVA2_3_NESTED_IF_CALIFAS.cpp
#include <iostream>
#include <sstream>
#include <string>
#include <cstdlib>
#include <cmath>

using namespace std;

// Headers
string toString (double);
int toInt (string);
double toDouble (string);

int main() {
    cout << "Introduce tu calificacion" << endl;
    int iCalificacion;
    
    cin >> iCalificacion;
    if (iCalificacion == 100) {
        cout << "A" << endl;
    } else {
        if (iCalificacion == 90) {
            cout << "B" << endl;
        } else {
            if (iCalificacion == 80) {
                cout << "C" << endl;
            } else {
                if (iCalificacion == 70) {
                    cout << "D" << endl;
                } else {
                    if (iCalificacion == 60) {
                        cout << "F" << endl;
                    } else {
                        cout << "Calificacion fuera de rango" << endl;
                    }
                }
            }
        }
    }
    return 0;
}

// The following implements type conversion functions.
string toString (double value) { //int also
    stringstream temp;
    temp << value;
    return temp.str();
}

int toInt (string text) {
    return atoi(text.c_str());
}

double toDouble (string text) {
    return atof(text.c_str());
}

File: LISTADO DE PRÁCTICAS DE LA SEGUNDA EVALUACIÓN PARCIAL - LUIS ARTURO MONSIVAIS VARGAS - 18550667/TRABAJOS EN FLOWGORITH - JAVA - C++ - PYHTON/Códigos en Phyton/EVA2_3_NESTED_IF_CALIFAS.py
print("Introduce tu calificacion")
iCalificacion = int(input())
if iCalificacion == 100:
    print("A")
else:
    if iCalificacion == 90:
        print("B")
    else:
        if iCalificacion == 80:
            print("C")
        else:
            if iCalificacion == 70:
                print("D")
            else:
                if iCalificacion == 60:
                    print("F")
                else:
                    print("Calificacion fuera de rango")
